<launch>
  <arg  name="input_image"        default="/camera/rgb/image_raw"/>
  <!--
  <arg     if="$(env USB_CAMERA)" name="input_image"        default="/camera/rgb/image_raw"/>
  <arg unless="$(env USB_CAMERA)" name="input_image"      default="/camera/color/image_raw"/>
  -->

    <!-- Detection node -->
    <node pkg="clbrobot_line_follower" name="clbrobot_line_detect" type="clbrobot_line_node" cwd="node" output="screen">
  	<param name="max_speed" type="double" value="0.2" />
  	<!-- yellow: 0,  red: 1, green: 2, blue:3-->
  	<param name="color" type="int" value="0" />
  	<param name="detect_mode" type="bool" value="false" />
  	<param name="servo_ctrl" type="bool" value="false" />
        <param name="image_topic" value="$(arg input_image)" />

    </node>

</launch>
